Modules/clinic/arraymodule.c.h

```diff
@@ -359,7 +359,7 @@ PyDoc_STRVAR(array_array_frombytes__doc__,
 "frombytes($self, buffer, /)\n"
 "--\n"
 "\n"
-"Appends items from the string, interpreting it as an array of machine values, as if it had been read from a file using the fromfile() method).");
+"Appends items from the string, interpreting it as an array of machine values, as if it had been read from a file using the fromfile() method.");
```

Comment:
This file is automatically generated from arraymodule.c.  It should not be edited by hand?  Did you, are did you run 'make regen' (I believe, on *nix) to regenerate it?  Or are you using windows?  I ask because something about this file caused the error.